docs/README.md

```diff
@@ -184,3 +184,45 @@ except recurly.errors.NotFoundError as e:
     response = e.error.get_response()
     print("Give this request id to Recurly Support: " + response.request_id)
 ```
+
+### Webhooks
+
+Recurly can send webhooks to any publicly accessible server.
+When an event in Recurly triggers a webhook (e.g., an account is opened),
+Recurly will attempt to send this notification to the endpoint(s) you specify.
+You can specify up to 10 endpoints through the application. All notifications will
+be sent to all configured endpoints for your site. 
+
+See our [product docs](https://docs.recurly.com/docs/webhooks) to learn more about webhooks
+and see our [dev docs](https://dev.recurly.com/page/webhooks) to learn about what payloads
+are available.
+
+Although our API is now JSON, our webhook payloads are still formatted as XML for the time being.
+This library is not yet responsible for handling webhooks. If you do need webhooks, we recommend using a simple
+XML to dict parser. We recommend using a small dependency such as [xmltodict](https://github.com/martinblech/xmltodict).
```

Comment:
Could probably fold these last two sentences together like on the other PR but not a blocker.